cfgov/unprocessed/apps/filing-instruction-guide/js/fig-toc-frame.js

```diff
@@ -0,0 +1,32 @@
+export function buildMobileTOC() {
+    expandableTOCFrame = `<div class="o-expandable
+    o-expandable__padded
+    o-expandable__background
+    o-expandable__border">
+<button class="o-expandable_header o-expandable_target"
+    title="Expand Table Of Contents">
+<h3 class="h4 o-expandable_header-left o-expandable_label">
+    Table of Contents
+</h3>
+<span class="o-expandable_header-right o-expandable_link">
+    <span class="o-expandable_cue o-expandable_cue-open">
+        <span class="u-visually-hidden-on-mobile">Show</span>
+        {% include icons/plus-round.svg %}
+    </span>
+    <span class="o-expandable_cue o-expandable_cue-close">
+        <span class="u-visually-hidden-on-mobile">Hide</span>
+        {% include icons/minus-round.svg %}
+    </span>
+</span>
+</button>
+<div class="o-expandable_content toc-expandable_content">
+</div>
+</div>`
+
```

Comment:
Have you looked into how iregs templates work? They achieve the functionality you're looking for by duplicating the [default secondary nav template](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html) and [tweaking it](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/regulations3k/jinja2/regulations3k/regulations3k-secondary-navigation.html) for their needs. The expandables code is just the standard HTML without having to do any funky JS work.

Reply:
I have but I am equally confused by the approach present there.  They appear to be doing something with `.js`, as indicated [here](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/apps/regulations3k/js/index.js).  I would love to have more of a discussion on this Monday when you are back, plz plz plz...